add-chain/main.go

```diff
@@ -230,3 +261,32 @@ func getGasPayingToken(l1rpcURl string, SystemConfigAddress superchain.Address)
 
 	return (*superchain.Address)(&result.Addr), nil
 }
+
+func copyDeployConfigFile(sourcePath string, targetDir string) error {
+	data, err := os.ReadFile(sourcePath)
+	if err != nil {
+		return err
+	}
+	return os.WriteFile(path.Join(targetDir, "deploy-config.json"), data, os.ModePerm)
+}
+
+func writeGenesisValidationMetadata(commit string, targetDir string) error {
+	// Define default metadata params:
+	// These may not be sufficient to make the genesis validation work,
+	// but we address that with some manual trial-and-error intervention
+	// involving OPLabs engineers after the add-chain command runs.
+	const defaultNodeVersion = "18.12.1"
+	const defaultMonorepoBuildCommand = "pnpm"
+	const defaultGenesisCreationCommand = "opnode1"
```

Comment:
Is there any explanation about what `opnode1` and `opnode2` mean?

Reply:
I should add some. Essentially, the genesis creation command was previously entirely handled by an op-node subcommand, and there were (at least) two different APIs to that command (with respect to command line args). More recently, the command also involves using a `foundry` script, so I anticipate we may need to add a `foundry` option to this file.